### ISSUE TITLE
Generic link to my products and my account

### ISSUE THREAD

**Issue by CharlesNepote** [openfoodfacts/openfoodfacts-server#3170]
We need to communicate the link related to people own product contributions: useful for communication, documentation, etc.

Today we must know the user's id to build this link. Example:
https://world.openfoodfacts.org/editor/charlesnepote

We should have a generic link, such as:
https://world.openfoodfacts.org/editor/me or https://world.openfoodfacts.org/myproducts

It should work for:
* my contributions: https://world.openfoodfacts.org/contributor/me
* my edits: https://world.openfoodfacts.org/editor/me
* my pictures: https://world.openfoodfacts.org/photographer/me

Of course, that should work with the other facets:
https://world.openfoodfacts.org/contributor/me/state/to-be-completed

I'm not sure if it needs to be a real page or a redirect. As a first step, a redirect could be a good solution.
https://world.openfoodfacts.org/editor/me would redirect to https://world.openfoodfacts.org/editor/charlesnepote in my case.

**Comment by stephanegigandet** [comment]
It's better if it's a redirect I think.

When the user is not identified, it should redirect to /cgi/session.pl which should then redirect back to /editor/me (which after that will redirect to /editor/charlesnepote )

**Comment by areeshatariq** [comment]
Would like to work on this

**Comment by VaiTon** [comment]
@areeshatariq off you go! 👍

**Comment by hangy** [comment]
Be careful: The username `me` sems to be registered and in use. https://world.openfoodfacts.org/editor/me

**Comment by stephanegigandet** [comment]
@hangy : good point. Let's just take /editor/user instead of me. It's not taken.

**Comment by aleene** [comment]
What about creating a statistics page for the user where this data is presented?

**Comment by CharlesNepote** [comment]
I'd rather using /me which make more sense than /user. Would be difficult to change the "/me" account? I can write to this user account to tell him we have to delete his account but he can recreate one (he only modified one product a year ago).

**Comment by hangy** [comment]
We could also use `https://world.openfoodfacts.org/me`. There's currently no site at that URL.

**Comment by VaiTon** [comment]
> 
> 
> We could also use `https://world.openfoodfacts.org/me`. There's currently no site at that URL.

I think that URL should be used to create some kind of dashboard more than what we display now in `/editor/`.

> 
> 
> It's better if it's a redirect I think.
> 
> When the user is not identified, it should redirect to /cgi/session.pl which should then redirect back to /editor/me (which after that will redirect to /editor/charlesnepote )

Also this. If it isn't a redirect then, for example, it can't be shared to someone as a link to my profile.

**Comment by CharlesNepote** [comment]
> > We could also use `https://world.openfoodfacts.org/me`. There's currently no site at that URL.
> 
> I think that URL should be used to create some kind of dashboard more than what we display now in `/editor/`.

 +1

> > It's better if it's a redirect I think.
> 
> Also this. If it isn't a redirect then, for example, it can't be shared to someone as a link to my profile.

There is a downside: many people wil think they send you *their* products when sending https://world.openfoodfacts.org/contributor/me

IMHO https://world.openfoodfacts.org/contributor/me is only suited when:
* you don't know the user_id
* you're speaking to many people at the same time

In other cases https://world.openfoodfacts.org/contributor/[user_id] should be used.

**Comment by github-throwaway** [comment]
👋 Hi @areeshatariq, it has been over a week since you were assigned to this issue, but we haven't detected a linked pull request referencing this issue. Please link your PR (it can be a draft) at your earliest convenience. If you have any remaining questions, don't hesitate to ask ☺️

**Comment by github-throwaway** [comment]
@hangy @CharlesNepote @aleene @stephanegigandet @VaiTon 

This issue is 5 years old. Do we still consider it relevant? If yes, do we still implement it as it was proposed 5 years ago?
